src/main/java/net/gegy1000/plasmid/block/CustomBlock.java

```diff
@@ -0,0 +1,92 @@
+package net.gegy1000.plasmid.block;
+
+import com.google.common.base.Preconditions;
+import net.gegy1000.plasmid.registry.TinyRegistry;
+import net.minecraft.text.Text;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.BlockPos;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class CustomBlock {
+    private static final TinyRegistry<CustomBlock> REGISTRY = TinyRegistry.newStable();
+    private static final Map<BlockPos, CustomBlock> BLOCK_MAP = new HashMap<>();
+
+    private final Identifier id;
+    private final Text name;
+
+    private CustomBlock(Identifier identifier, Text name) {
+        this.id = identifier;
+        this.name = name;
+    }
+
+    public static Set<Identifier> getKeys() {
```

Comment:
```suggestion
    public static Set<Identifier> getIds() {
```

Reply:
if we are fixing that we should fix it in Custom Items

Reply:
Hmm alright, put this change in and we'll change it there too